_posts/blog/2018-11-12-Latin@s-in-Ohio-review

```diff
@@ -0,0 +1,76 @@
+Latinxs, at the same time presenting counternarratives to deconstruct the stereotypes of Latinxs in this country and, more 
+specifically, in Ohio. The collection, currently, consists of sixty-seven videos that range in length from twenty minutes to 
+two hours of participants answering various questions, developed by scholars. Participants’ backgrounds are vastly 
+differently; some have been in the state of Ohio since the 1940’s and others are recent arrivals. Also, their careers, 
+socioeconomic backgrounds, and reasons for moving to Ohio vary. The interviews will have one individual at a time, while 
+others will have multiple participants, including families, groups of friends, and couples. Because of the diversity of the 
+interviewees, their responses provide an intergenerational perspective on life in Ohio, as well as strategies to preserve 
+their culture (Foulis, 2018). 
+
+As I navigated the collect and the videos, my intentions were to review the Oral Narrative collection, solely as a scholar, 
+but as a fellow Midwestern Latina, I felt a strong connection to everyone’s stories. In the late 90s, my family moved from the 
+east coast to a city that is less than a few miles from Ohio. Many of the places mentioned in the videos, I recognized due to 
+my family’s frequent visits to these locations. To say the least, I was thrilled to engage with this material. 
+
+Coming from an multi-ethnic and biracial family, and moving from New York to the Midwest was a challenge for my family. We 
+left everything we were accustomed to, for a place where we did not see or hear people like us, nor did most people understand 
+our backgrounds. Therefore, as I watched the videos, I found myself agreeing with several of the comments made of 
+participants’ lived experiences and values, because to me it represented my families and our reality in the Midwest at a 
+specific moment in time. As a result, I felt connected to all of the individuals and proud to hear their stories. 
+           
+In addition, I was excited to find the majority of narratives were in Spanish, which was surprising due to the fact that 
+English is the dominant language in the Midwest. This important decision represented Dr. Foulis’ centering and celebration of 
+the Spanish language and Latinx culture, which was emphasized by the lack of English translations for the videos and shown by 
+the heritage speakers proudly using the language they are most comfortable with. This collection of video narrative is an 
+authentic example how the archive can bring to the forefront the voices of those usually excluded and marginalized to support 
+their efforts to preserve their stories.  
+
+Returning to my reactions as I read, I couldn’t help, but question my excitement. Why was I drawn to these stories, even 
+though I frequently and intentionally visited and engaged with Latinxs spaces? After reading “ To Suddenly Discover Yourself 
```

Comment:
You can make a stronger argument here by taking ownership over what Caswell et al have written. Instead of narrating your experience of reading the article, try claiming its argument for yourself: "Returning t....latinxs spaces? I found the answer to my questions in the concepts of symbolic annihilation and representational belonging described by Caswell et al. in "To Suddenly Find Yourself Existing." Symbolic annihilation is defined as...." [and then you can continue as normal.]